internal/config/config.go

```diff
@@ -93,6 +94,8 @@ type AgentConfig struct {
 	// Quiet is the option for running in quiet mode
 	Quiet    bool
 	Hostname string
+
+	Enterprise enterprise.Config
```

Comment:
So for this to be activated, the user would need an `enterprise` table underneath the `[agent]` table? What does that look like? Should we put a commented-out example in the sample config?